packages/SelectorCounter/SelectorCounter.md

```diff
@@ -1,15 +1,21 @@
 ## Minimal usage
 
 ```
-<SelectorCounter />
+<Box between={2}>
+  <label htmlFor="docs_example-1" className="docs_selcounter-label">Simple example</label>
+  <SelectorCounter id="docs_example-1" />
```

Comment:
Is there a reason why we're using `id` attributes for test selectors and not `data-test`?

Reply:
We can't link the label and selector counter input using `data-test-id`, `htmlFor` looks for an id attribute.

Reply:
@theetrain This is a markdown file, not a test. :)

This id is only used for the example. Because the SelectorCounter does not have a `label` element like our other form fields (Input, Select, etc), it needs to be properly associated with the form element.

Reply:
I see, I totally missed the context here 😲